editoast/src/views/path/pathfinding.rs

```diff
@@ -318,9 +318,10 @@ async fn pathfinding_blocks_batch(
                 to_compute_hashes.push(hash);
             }
             Err(result) => {
+                let arc_result = Arc::new(result.clone());
```

Comment:
Do you need to clone here?


```suggestion
                let arc_result = Arc::new(result);
```

Reply:
It's used again l.325